src/Bundle/Admin/Resources/views/SkRecherche/details.html.twig

```diff
@@ -5,17 +5,28 @@
 {% set menu_user = true %}
 
 {% block body %}
-    <div class="card mb-4">
-        <div class="card-header text-center">
-            A propos de {{ users }} du {{ users.etsNom }}
-        </div>
-        <div class="card-body">
-            <ul>
-                <li>NOM : {{ users.usrLastname }}</li>
-                <li>PRÉNOM : {{ users.usrFirstname }}</li>
-                <li>ADRESSE : {{ users.usrAddress }}</li>
-                <li>CONTACT : {{ users.usrPhone }}</li>
-            </ul>
+    <div class="row">
```

Comment:
Should use directly app.user instead of creating new object.
At least, use singular name because it is a singular user